File: app/src/main/java/starbright/com/projectegg/data/local/model/response/ExtendedIngredientResponse.java
/**
 * Created by Andreas on 15/8/2018.
 */

package starbright.com.projectegg.data.local.model.response;

import com.google.gson.annotations.SerializedName;

public class ExtendedIngredientResponse {

    @SerializedName("id")
    private String mId;

    @SerializedName("name")
    private String mName;

    @SerializedName("amount")
    private int mAmount;

    @SerializedName("unit")
    private String mUnit;

    @SerializedName("image")
    private String mImage;

    public String getId() {
        return mId;
    }

    public String getName() {
        return mName;
    }

    public int getAmount() {
        return mAmount;
    }

    public String getUnit() {
        return mUnit;
    }

    public String getImage() {
        return mImage;
    }
}

File: app/src/main/java/starbright/com/projectegg/data/model/response/ExtendedIngredientResponse.kt
/**
 * Created by Andreas on 15/8/2018.
 */
package starbright.com.projectegg.data.model.response

import com.google.gson.annotations.SerializedName

data class ExtendedIngredientResponse (
    @SerializedName("id")
    val id: String? = null,

    @SerializedName("name")
    val name: String? = null,

    @SerializedName("amount")
    val amount: Float = 0f,

    @SerializedName("unit")
    val unit: String? = null,

    @SerializedName("image")
    val image: String? = null
)